docs/source/create/tools.rst

```diff
@@ -45,11 +45,11 @@ When you select an entity in the Entity List, you can:
 	Shapes: Create in VR
 	------------------------
 
-	The Shapes app is a Marketplace app that lets you :doc:`add and edit basic shapes (entities) <entities/create-entities>` in Vircadia. You can use Shapes only in VR mode. 
+	The Shapes app is an app that lets you :doc:`add and edit basic shapes (entities) <entities/create-entities>` in Vircadia. You can use Shapes only in VR mode. 
 
 	To get the Shapes app: 
 
-	1. In Interface, pull up your HUD or Tablet and go to **Market**. 
+	1. In Interface, pull up your HUD or Tablet and go to **More**. 
```

Comment:
The Shapes app isn't actually in the more app. I created an issue for this https://github.com/vircadia/vircadia-docs-sphinx/issues/165